packaging/systemd/firewalld-no-iptables.conf

```diff
@@ -0,0 +1,10 @@
+[Service]
+InaccessiblePaths=/usr/sbin/xtables-nft-multi
+
+# This is a workaround for https://issues.redhat.com/browse/NP-641
+# it will trigger some warnings on the selinux audit log, but it's much
+# better than the failure mode of having all iptables flushed by
+# firewalld
+
+# hopefully in the future we can have a setting in firewalld to avoid
+# iptables flushing when using nft.
```

Comment:
We need to rephrase this explanation. I would not include a reference to the issue, which is currently private. Also, we should only explain what we're trying to solve without future statements.

Reply:
We have talked about this in the past, but IMHO, as a project maintainer, it always helped me to figure out why certain things were done in projects, it is true that you can always look at the code/commit and find the original reference. But for example it gets harder when the file has moved around and the git reference becomes harder to track.

Reply:
This file will end up on user systems, so we need to review the text in the comments, like documentation. Let's discuss it further and reach a conclusion backed up by our content experts.